source/dataaccess.rst

```diff
@@ -0,0 +1,27 @@
+============================================
+Data Access for advanced use
+============================================
+
+-------------------------------------------------------------
+Functionality
+-------------------------------------------------------------
+* Users should be able to produce a dataset and upload to the data access service without time-consuming transformations.
+* Users must be able to download a copy of the entire dataset.
+* Users must be able to stream and filter parts of the dataset.
+* Users need access to enough metadata to be able to locally post-process, reproject etc. the dataset.
+* Data access service need to support the requirements for DOI: <ref>.
```

Comment:
jeg tror ikke DOI har spesielle krav ang tilgang. Sjekk f.eks. datatype "Withdrawn"

Reply:
Jeg tenkte mest her på om datasettet som urlen referer til kan endre seg eller ikke. Altså om data access -tjenestene må kunne fryse datasett. Evt. om enkelte datasett er statiske, mens andre igjen fylles på hele tiden.

Reply:
url/doi viser til en landingsside for hvert datasett. Vi velger selv hvordan vi definerer datasettene. Når det gjelder f.eks. satellitt- eller modelldata, så gir det i mitt hode lite mening å tilordne doi for hver akvisering eller simulering, men heller ha et "parent" datasett som henviser til underdatasett i et parent-child forhold. Alle datasett må uansett ha en unik ID, men jeg tenker det er unødvendig at alle har doi. Men dette er ganske enkelt å endre seinere, hvis det skulle være behov for det. DOI-tildelingen krever såvidt jeg husker at vi følger metadatakravene til DataCite: https://schema.datacite.org/, men så lenge vi fyller inn MMD-filene korrekt, så går det av seg selv.